src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$font-family: Roboto, sans-serif;
```

Comment:
I don't quite understand why you need another variable that you don't use with the same font.